lib/minio/client.rb

```diff
@@ -77,6 +77,10 @@ def admin_policy_remove(policy_name)
     response.status
   end
 
+  def get_presigned_url(method, bucket_name, object_name, expires: 3600, request_date: Time.now.utc)
```

Comment:
does this `request_date` allow creating a URL from an earlier time or a URL that will be valid in future? I guess the latter is OK but the first one, not so much. Also, that functionality doesn't exist in UI. Should we simply move `request_date` into the function and not allow in the API?

Reply:
I used the same parameters with the Python library. If you believe it's unnecessary, I can remove it.

Reply:
Let's move it into thw function for now. If we need a url to be valid in feature, we can always move it back. I don't see a reason to introduce it into the API, for now.

Reply:
I moved it